analyzers/src/SonarAnalyzer.Common/DiagnosticAnalyzer/SonarDiagnosticAnalyzer.cs

```diff
@@ -18,17 +18,36 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System;
 using Microsoft.CodeAnalysis.Diagnostics;
 
 namespace SonarAnalyzer.Helpers
 {
     public abstract class SonarDiagnosticAnalyzer : DiagnosticAnalyzer
     {
+        private const string EnableConcurrentProcessing = "SONAR_DOTNET_ENABLE_CONCURRENT_PROCESSING";
+        protected virtual bool ConcurrentProcessingDisabled => IsConcurrentProcessingDisabled();
```

Comment:
Is there a strong reason for this negation? `true` turns something off and that's confusing brain twist.

I'd strongly prefer `ConcurrentProcessingEnabled` with default value of `true` and turning it off with `false`. That is also aligned with the environment variable name.

Reply:
I'd actually prefer the original `EnableConcurrentExecution ` as it was specified to be aligned with Roslyn API and clear about it's action.

Reply:
Done here - https://github.com/SonarSource/sonar-dotnet/pull/4708